src/EventStore.Client.Operations/EventStoreOperationsClient.cs

```diff
@@ -18,9 +21,17 @@ public partial class EventStoreOperationsClient : EventStoreClientBase {
 		private readonly Operations.Operations.OperationsClient _client;
 		private readonly ILogger _log;
 
+		/// <summary>
+		/// Constructs a new <see cref="EventStoreOperationsClient"/>. This method is not intended to be called directly in your code.
```

Comment:
Should we not provide more guidance around which method is supposed to be used?

Reply:
This constructor would be called by DI so unfortunately it needs to be public